android/src/main/java/io/wifi/p2p/WiFiP2PManagerModule.java

```diff
@@ -64,6 +64,25 @@ public void onConnectionInfoAvailable(WifiP2pInfo wifiP2pInformation) {
         });
     }
 
+   @ReactMethod
+    public void getGroupPassphraseInfo(final Promise promise) {
+         manager.requestGroupInfo(channel, new WifiP2pManager.GroupInfoListener() {
+            @Override
+            public void onGroupInfoAvailable(WifiP2pGroup group) {
+                System.out.println(group);
+
+                if (group != null)
+                {
+                  groupPassword = group.getPassphrase();
+                  promise.resolve(groupPassword);
+                }
+                else {
+                    promise.resolve("NOT_AVAILABLE");
```

Comment:
Maybe do we need to pass a `null` instead of string constant?
What if user created a password `NOT_AVAILABLE? 😈